### PR TITLE
Fix the bad type evaluated from AttributeExpression on vertices

### LINKED ISSUE #5153: Unexpected query error
**Please check the FAQ documentation before raising an issue**

**Describe the bug (__required__)**

Look at the queries below:

```txt
(root@nebula) [gdlancer]> MATCH p0 = (v0)-[e0]->()-[e1]->() WHERE (id(v0) == 4) AND ((NOT e0.Rel_4_0_Bool)) UNWIND nodes(p0) AS ua0 WITH ua0, e0  WITH e0, ua0 WHERE ua0.Label_6.Label_6_1_Bool return id(ua0)
[ERROR (-1005)]: Failed to evaluate condition: $ua0.Label_6.Label_6_1_Bool. For boolean conditions, please write in their full forms like <condition> == <true/false> or <condition> IS [NOT] NULL.

Wed, 28 Dec 2022 15:09:58 CST

(root@nebula) [gdlancer]> MATCH p0 = (v0)-[e0]->()-[e1]->() WHERE (id(v0) == 4) AND ((NOT e0.Rel_4_0_Bool)) UNWIND nodes(p0) AS ua0 WITH ua0, e0  WITH e0, ua0  return ua0.Label_6.Label_6_1_Bool
+----------------------------+
| ua0.Label_6.Label_6_1_Bool |
+----------------------------+
| true                       |
| BAD_TYPE                   |
| BAD_TYPE                   |
| true                       |
| BAD_TYPE                   |
| false                      |
| true                       |
| BAD_TYPE                   |
| BAD_TYPE                   |
+----------------------------+
Got 9 rows (time spent 3.806ms/21.363541ms)

Wed, 28 Dec 2022 15:10:32 CST

(root@nebula) [gdlancer]> MATCH p0 = (v0)-[e0]->()-[e1]->() WHERE (id(v0) == 4) AND ((NOT e0.Rel_4_0_Bool)) UNWIND nodes(p0) AS ua0 WITH ua0, e0  WITH e0, ua0  return labels(ua0)
+--------------------------------------------------------------------+
| labels(ua0)                                                        |
+--------------------------------------------------------------------+
| ["Label_5", "Label_6", "Label_0"]                                  |
| ["Label_5"]                                                        |
| ["Label_1"]                                                        |
| ["Label_5", "Label_6", "Label_0"]                                  |
| ["Label_5"]                                                        |
| ["Label_5", "Label_6", "Label_3", "Label_4", "Label_0", "Label_1"] |
| ["Label_5", "Label_6", "Label_0"]                                  |
| ["Label_5"]                                                        |
| ["Label_1"]                                                        |
+--------------------------------------------------------------------+
Got 9 rows (time spent 3.539ms/20.568834ms)

Wed, 28 Dec 2022 15:11:11 CST
```

The first query return an unexpected error when there are vertexes without tag `Label_6`.

**Your Environments (__required__)**

* OS: `uname -a`
* Compiler: `g++ --version` or `clang++ --version`
* CPU: `lscpu`
* Commit id (e.g. `a3ffc7d8`) 6cb56c340 (community edition)

**How To Reproduce(__required__)**

1. Step 1
2. Step 2
3. Step 3

**Expected behavior**

**Additional context**

### PR DESCRIPTION
**Pull request by xtcyclist**
## What type of PR is this?
- [X] bug
- [ ] feature
- [ ] enhancement

#### Issue(s) number: 
Close https://github.com/vesoft-inc/nebula-ent/issues/2020
Close #5153 

## Checklist:
Tests:
- [ ] Unit test(positive and negative cases)
- [ ] Function test
- [ ] Performance test
- [ ] N/A

Affects:
- [ ] Documentation affected (Please add the label if documentation needs to be modified.)
- [ ] Incompatibility (If it breaks the compatibility, please describe it and add the label.）
- [ ] If it's needed to cherry-pick (If cherry-pick to some branches is required, please label the destination version(s).)
- [ ] Performance impacted: Consumes more CPU/Memory

## Release notes:

Please confirm whether to be reflected in release notes and how to describe:
> ex. Fixed the bug .....

### CODE REVIEW

**Comment by xtcyclist** [comment]
This PR depends on https://github.com/vesoft-inc/nebula/pull/5149.

File: tests/tck/conftest.py [hunk 1]
```diff
@@ -195,7 +195,7 @@ def preload_space(
     elif space == "ldbc_v0_3_3":
         exec_ctx["ldbc_v0_3_3"] = load_ldbc_v0_3_3
     elif space == "test":
-        exec_ctx["test"] = load_test_data
```
Comment (Shylock-Hg) [inline tests/tck/conftest.py:198]: Why change this?

File: src/common/expression/AttributeExpression.cpp [hunk 2]
```diff
@@ -99,6 +99,10 @@ const Value &AttributeExpression::eval(ExpressionContext &ctx) {
       result_ = time::TimeUtils::getDateTimeAttr(lvalue.getDateTime(), rvalue.getStr());
       return result_;
     default:
+      if (lvalue.isNull() && lvalue.getNull() == NullType::UNKNOWN_PROP) {
+        // return UNKNOWN_PROP as plain null values, instead of bad type.
+        return Value::kNullValue;
+      }
```
Comment (czpmango) [inline src/common/expression/AttributeExpression.cpp:105]: Better to modify the behivor of `isNull`.

**Review by Aiee — APPROVED** [review]

**Review by jievince — APPROVED** [review]

**Review by nevermore3 — APPROVED** [review]

**Review by codesigner — APPROVED** [review]

File: src/common/expression/AttributeExpression.cpp (hunk 2 above)
Comment (xtcyclist) [reply src/common/expression/AttributeExpression.cpp:105]: Using existing methods of Value for now.

**Pull request merged**